things/postgres/init.go

```diff
@@ -103,8 +103,8 @@ func migrateDB(db *sqlx.DB) error {
 				Id: "things_5",
 				Up: []string{
 					`CREATE TABLE IF NOT EXISTS groups (
-						id          VARCHAR(254) UNIQUE NOT NULL,
-						owner_id    VARCHAR(254),
+						id          UUID,
```

Comment:
`id          UUID UNIQUE NOT NULL,`

Reply:
https://github.com/MainfluxLabs/mainflux/actions/runs/4223220554/jobs/7332693906